### DISCUSSION TITLE
Change Default Branch to develop

### DISCUSSION THREAD

**Discussion by rozyczko** [easyscience/.github#19] [Ideas]
Proposed:

Change the default branch on all projects from `master` to `develop`.
The default branch is considered the “base” branch in your repository, against which all pull requests and code commits are automatically made, unless you specify a different branch.

Most of the time, when we do PRs, they go to `develop`. Only during the release we merge into master.
Seeing how easy it is to make a mistake when creating a PR and base it on the default `master` branch, it would be better to have `develop` as the default.

This will save time on `git revert` :)

**Comment by andped10** [comment]
You should be careful.  I tried it ones and if I remember correctly Github then starts to display the content of the README etc on the Repo pages from the development branch.  Also, I think the badges for the builds etc, which we use on other pages then gets screwed up

**Comment by rozyczko** [comment]
From

https://stackoverflow.com/questions/44466618/default-branch-for-pull-request

"The downside is that this is also the cloning branch. It forces us to use the develop branch as our main branch. It also might interfere with some bots such as "wei/pull"

Considering this and what Andreas said above,  this would be a change breaking our workflow more than expected.

Apparently, just modifying the default target branch for PRs is not possible.

**Comment by rozyczko** [comment]
Therefore closing as outdated. We might revisit this once GH offers the functionality.
For the time, we just have to be careful with PRs, both creating and reviewing.
